code/modules/clothing/clothing.dm

```diff
@@ -95,7 +95,7 @@
 	// sigh, ok, so it's not ACTUALLY infinite nutrition. this is so you can eat clothes more than...once.
 	// bite_consumption limits how much you actually get, and the take_damage in after eat makes sure you can't abuse this.
 	// ...maybe this was a mistake after all.
-	food_reagents = list(/datum/reagent/consumable/nutriment = INFINITY)
+	food_reagents = list(/datum/reagent/consumable/nutriment/cloth = INFINITY)
```

Comment:
```suggestion
	food_reagents = list(/datum/reagent/consumable/nutriment/cloth = INFINITY, /datum/reagent/consumable/maltodextrin/microplastics = INFINITY)
```

Nutriment/cloth is implemented in an unnecessarily hacky way. There's no reason to add a bloat reagent just to force the distribution of reagents to be 7:3. A 1:1 distribution is fine - just tweak the values on the microplastics so it fits accordingly.

Numbers are suggested in a different comment.